packages/docs/page-config/ui-elements/menu/index.ts

```diff
@@ -0,0 +1,54 @@
+export default definePageConfig({
+  blocks: [
+    block.title("Menu"),
+    block.paragraph("The `VaMenu` component is used to display structured information under the menu button."),
+
+    block.link("VaMenuList", "/ui-elements/menu-list", { preText: "See " }),
+
+    block.subtitle("Examples"),
+
+    block.example("Default", {
+      title: "Basic usage",
+      description: "Required to have a anchor. Anchor can be anything, component will automatically listen for clicks, keyboard navigation and open/close menu. To show menu items you can use `options` prop or `VaMenuItem` component.",
+    }),
+    block.example("MenuItem"),
```

Comment:
Let's add description for this component. Right now it's not clear why this example exists.

"You can also use slot and `VaMenuItem` component to achieve better flexibility:"